sdk/compiler/damlc/tests/src/DA/Test/DamlcIntegration.hs

```diff
@@ -329,7 +330,10 @@ getIntegrationTests registerTODO scenarioService (packageDbPath, packageFlags) =
                     , dlintHintFiles = NoDlintHintFiles
                     }
                 , optPackageImports = packageFlags
-                , optUpgradeInfo = (optUpgradeInfo opts0) { uiWarnBadInterfaceInstances = True, uiWarnBadExceptions = True }
+                , optDamlWarningFlags =
+                    [ RawDamlWarningFlag upgradeInterfacesName AsWarning upgradeInterfacesFilter
+                    , RawDamlWarningFlag upgradeExceptionsName AsWarning upgradeExceptionsFilter
```

Comment:
Maybe pull these out into values in the module that defines them, it'd be easy to accidentally use the wrong filter with the given name

Reply:
Done